k8s-openapi-codegen-common/src/swagger20/definitions.rs

```diff
@@ -88,6 +88,16 @@ impl<'de> serde::Deserialize<'de> for RefPath {
 	}
 }
 
+#[cfg(all(feature = "serde", feature = "schema"))]
+impl serde::Serialize for RefPath {
+	fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+	where
+		S: serde::Serializer,
+	{
+		serializer.serialize_str(&format!("#/definitions/{}", &self.path))
```

Comment:
`format_args!("#/definitions/{}", &self.path).serialize(serializer)` will do it without allocating a temporary `String`, since `std::fmt::Arguments` impls `serde::Serialize` using `serde::Serializer::collect_str`